### PR TITLE
Fix build

### PR DESCRIPTION
**Pull request by szabikr**
## Description

This might be the fix for build.

### CODE REVIEW

**Review by TomBillingtonUK — APPROVED** [review]

File: package-lock.json [hunk 1]
```diff
@@ -25874,9 +26138,9 @@
...
         "is-glob": {
-          "version": "4.0.0",
-          "resolved": "https://registry.npmjs.org/is-glob/-/is-glob-4.0.0.tgz",
-          "integrity": "sha1-lSHHaEXMJhCoUgPd8ICpWML/q8A=",
+          "version": "4.0.1",
+          "resolved": "https://registry.npmjs.org/is-glob/-/is-glob-4.0.1.tgz",
+          "integrity": "sha512-5G0tKtBTFImOqDnLB2hG6Bp2qcKEFduo4tZu9MT/H6NQv/ghhy30o55ufafxJ/LdH79LLs2Kfrn85TLKyA7BUg==",
           "dev": true,
           "requires": {
             "is-extglob": "^2.1.1"
```
Comment (rossfauldsbjss) [inline package-lock.json:26146]: Lots of version changes in the package lock - is this intended?

**Review by rossfauldsbjss — CHANGES_REQUESTED** [review]
Just want to make sure  all the package-lock changes are intentional for this fix.

**Comment by andrew-lai-bjss** [comment]
There has been an update to the package and package-lock file in develop so don't expect this will merge in now.

**Comment by szabikr** [comment]
This is not the most appropriate fix

**Pull request closed without merging**